ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -671,6 +692,47 @@ VAV_SD_DTM:
   - SD
   - DTM
 
+VAV_DSP_DFSS_DFVSC:
+  guid: "9f4a0f1a-a6e5-4481-bf7c-bd35dcbe2258"
+  description: "Fan powered VAV with variable speed control."
+  is_canonical: true
+  implements:
+  - VAV
+  - DSP
+  - DFSS
+  - DFVSC
+
+VAV_DSP_HTZC_DFSS_DFVSC:
+  guid: "2d73d3ca-836b-43e2-b474-5262a9cf5e52"
+  description: "Fan powered VAV with variable speed control."
```

Comment:
Make this description better - how is this different from the one above

Reply:
Done